zerver/views/video_calls.py

```diff
@@ -47,6 +47,15 @@ def __init__(self) -> None:
         super().__init__(_("Invalid Zoom access token"))
 
 
+def parse_jitsi_server_url(
+    value: str, special_values_map: Mapping[str, Optional[str]]
+) -> Optional[str]:
+    if value in special_values_map:
+        return special_values_map[value]
+
+    return value
```

Comment:
Is there a reason for this to be in this module? I would think it should just sit above the function that uses it.

Reply:
I don't exactly remember, maybe I confused the module with `lib/video_calls.py`. Moving it sounds good to me. I have updated the PR accordingly.